### PR TITLE
Example implementation, mostly untested

### PR DESCRIPTION
**Pull request by Gustorn**
I still ended up doing my example as well, take a look at it if you'd like.

### CODE REVIEW

File: MD/md_gustorn.cpp [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+/*----------------------------------------------------------------------------//
+* STRUCTS AND FUNCTIONS
+*-----------------------------------------------------------------------------*/
+
+// Holds our data in a central struct, to be called mainly in a vector
+struct Particle {
+    Point pos;
+    Vector vel;
+    uint64_t pid;
```
Comment (Gustorn) [inline MD/md_gustorn.cpp]: `uint64_t` is better for indexes, you don't need negative numbers so this adds support for basicly an arbitrary number of  particles.

File: MD/md_gustorn.cpp [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+inline bool operator<(const Interaction& lhs, const Interaction& rhs) {
+    return lhs.rtime < rhs.rtime;
+}
+
+// Makes starting data
+std::vector<Particle> populate(uint64_t pnum, double box_length, double max_vel);
+
+// Makes the interactions for our simulation later, required starting data
+void update_interactions(const std::vector<Particle>& particles,
```
Comment (Gustorn) [inline MD/md_gustorn.cpp]: I merged the two separate update functions, they did the same thing anyway.

**Comment by Gustorn** [comment]
Just an other note: this compiles with no warnings with `clang++ -std=c++11 -Wno-c++98-compat`.

**Comment by leios** [comment]
Hey, man! Thanks a lot (again). I am going to be looking through these just before stream today (so only for a few more minutes unfortunately). I have added it as another auxiliary file; however, as I told Toeger, I think I need to use my code for today and leave examples of other code that is written in a new / different way. I am afraid that if I ended up using someone else's code, some of the viewers who are not so coding savvy might get a little lost.

Thanks again for showing me another way to do this. It's super helpful! =)

**Pull request merged**